packages/cli/internal/pkg/aws/cdk/progress.go

```diff
@@ -21,13 +22,21 @@ type ProgressEvent struct {
 	StepDescription string
 	Outputs         []string
 	Err             error
+	ExecutionName   string
+	LastOutput      string
+}
+
+type Result struct {
+	ExecutionName string
+	Outputs       []string
+	Err           error
 }
 
 func (p ProgressStream) DisplayProgress(description string) error {
 	var lastEvent ProgressEvent
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
-	barReceiver := runProgressBar(ctx, description)
+	barReceiver := runProgressBar(ctx, description, 1)
```

Comment:
This is a bit hard to read. From the code, it's not clear what that magic number "1" means. It doesn't block the PR, but I would much appreciate, if you consider adding a bit of clarity:

```go
...
numberOfChannels := 1
barReceiver := runProgressBar(ctx, description, numberOfChannels)
...
```